src/indicators/flooding/floodscan/floodscan.py

```diff
@@ -3,7 +3,10 @@
 import geopandas as gpd
 from rasterstats import zonal_stats
 import xarray as xr
-import rioxarray
+
+# not using rioxarray directly but using .rio
+# so let flake8 ignore it
+import rioxarray  # noqa
```

Comment:
Why did you use `# noqa: F401` in the other file and not here?

Reply:
Laziness😇 changed now